src/screens/SearchScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { ScrollView } from 'react-native';
```

Comment:
Can you run `yarn prettier --write .` in terminal to fix this?

Reply:
I tried this and got "error Command failed with exit code 1." and " Usage: prettier [options] [file/dir/glob ...]

By default, output is written to stdout.
Stdin is read if it is piped to Prettier and no files are given." 

how do I fix that?